src/main.ts

```diff
@@ -187,7 +187,7 @@ class ReasonMLLanguageClient extends AutoLanguageClient {
   }
 
   startRls(projectPath: string) {
-    const serverPath = require.resolve(`../rls/rls-${process.platform}-${RLS_VERSION}.exe`)
+    const serverPath = require.resolve(`../rls/${this.getRlsBin()}`)
```

Comment:
Un-renaming files since it's quite annoying to rename 3 files every time I bump RLS.